patches/minecraft/net/minecraft/client/renderer/GameRenderer.java.patch

```diff
@@ -39,12 +39,7 @@
        p_228378_4_.func_227863_a_(Vector3f.field_229179_b_.func_229187_a_(activerenderinfo.func_216777_e()));
        p_228378_4_.func_227863_a_(Vector3f.field_229181_d_.func_229187_a_(activerenderinfo.func_216778_f() + 180.0F));
        this.field_78531_r.field_71438_f.func_228426_a_(p_228378_4_, p_228378_1_, p_228378_2_, flag, activerenderinfo, this, this.field_78513_d, matrix4f);
-+      this.field_78531_r.func_213239_aq().func_219895_b("forge_render_last");
-+      net.minecraftforge.client.ForgeHooksClient.dispatchRenderLast(this.field_78531_r.field_71438_f, p_228378_4_, p_228378_1_, matrix4f, p_228378_2_);
```

Comment:
I am slightly unsure if this would constitute a breaking change or not, given at least for fast/fancy (where this *does* work), any mod that is currently listening to the existing RenderLastEvent would stop having it fired and I believe have their code get "broken". So we may need to leave this in, or if the new render last event is equivalent in terms of when/how it fires, have the new one still just fire the old event instead of the new one until 1.17

Reply:
The new RenderWorldLastEvent is a 1:1 replacement, will take a look at how to merge them - possibly can fire the old one using the new firing method but projectionMatrix in the old event would have to be null (I doubt anyone managed to use it except me after I've added it) other parameters are available, but for sure we can fire both of them until 1.17

Reply:
My bad - all arguments are available, event fully revived and fired after the new one.